okhttp/src/test/java/io/grpc/okhttp/TlsTest.java

```diff
@@ -117,6 +117,7 @@ public void mtls_succeeds() throws Exception {
     ManagedChannel channel = grpcCleanupRule.register(clientChannel(server, channelCreds));
 
     SimpleServiceGrpc.newBlockingStub(channel).unaryRpc(SimpleRequest.getDefaultInstance());
+    server.shutdown();
```

Comment:
What do these do? The Cleanup rule will shut down the server just after this line. Do we have to shut down the server before the client?

It isn't like OkHttpServer.acceptConnecitons() is spinning tightly in its loop. `accept()` blocks until there is a connection.

Reply:
But because the client shuts down before the server, after it falls out of the accept it goes through the logic that is skipped when the shutdown flag is set.

Reply:
Darn reality not matching the theory .... :(